tray.c

```diff
@@ -45,12 +45,41 @@ HMENU hMenu;
 HMENU hMenuConn[MAX_CONFIGS];
 HMENU hMenuService;
 
+HBITMAP hbmpConnected;
```

Comment:
This is not used with `SetMenuItemBitmaps()`. We only assign it in `CreateBitmaps()`.

Reply:
> This is not used with `SetMenuItemBitmaps()`. We only assign it in `CreateBitmaps()`.

Right, I had considered using that image in place of the default check mark when connected. Removed & fixed the spacing errors (tabs).
Thanks.